models/spleen_ct_segmentation/configs/inference.json

```diff
@@ -100,7 +100,7 @@
         "sw_batch_size": 4,
         "overlap": 0.5
     },
-    "postprocessing": {
+    "postprocess": {
```

Comment:
do this change to see if we can get expected error.
Error happens: https://github.com/Project-MONAI/model-zoo/actions/runs/10039656632/job/27744051207?pr=602#step:7:731
cc @MingxueG